src/createServer.js

```diff
@@ -1,3 +1,37 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { validate } = require('./validate');
+
+const createServer = () => {
+  return http.createServer((req, res) => {
+    const url = new URL(req.url, `http://${req.headers.host}`);
+    const originalText = url.pathname.slice(1);
+    const targetCase = url.searchParams.get('toCase');
+
+    res.setHeader('Content-Type', 'application/json');
+
+    const errors = validate(originalText, targetCase);
+
+    if (errors.length > 0) {
+      res.statusCode = 400;
+      res.end(JSON.stringify({ errors }));
```

Comment:
The error response should include a `Bad request` status text. Consider adding `res.statusMessage = 'Bad request';` before ending the response to comply with the task requirements .